setup.py

```diff
@@ -286,26 +290,17 @@ def cmake():
         subprocess.check_call(cmd_str)
 
 
-def version_tag():
-    from tools.gen_nvfuser_version import get_version
-
-    version = get_version()
-    if OVERWRITE_VERSION:
-        version = version.split("+")[0]
-        if len(VERSION_TAG) != 0:
-            version = "+".join([version, VERSION_TAG])
-    return version
-
-
 def main():
-    if BUILD_SETUP:
-        cmake()
-
+    # NOTE(crcrpar): Deliberately build basically two dynamic libraries here so that they can
+    # be treated as "nvfuser_package_data". This function call will put the two of "nvfuser" and
+    # "nvfuser_codegen" into "./nvfuser/lib", and the former will be "nvfuser._C".
+    cmake()
```

Comment:
I think this will break when we try to run `python setup.py clean` on a setup where things are partially broken.

also, if we are running clean, we probably don't need to configure cmake at the same time.